interim_patch.py

```diff
@@ -0,0 +1,17 @@
+# fix setuptools.distutils import in torch
```

Comment:
Can we move this file to `tests/vso_tools`?

Reply:
moved